src/server/components/dashboard/relatedLinks/router.ts

```diff
@@ -0,0 +1,23 @@
+import express from "express";
+import { confirm, edit, handleRelatedLinkIndexParam } from "./handlers";
+
+export const relatedLinksRouter = express.Router();
+
+relatedLinksRouter.param("relatedLinkIndex", handleRelatedLinkIndexParam);
+
+relatedLinksRouter.use("*", (req, res, next) => {
+  res.locals.csrfToken = req.csrfToken();
```

Comment:
This adds the CSRF token to the render context, so it can be accessed via `{{csrfToken}}`. I believe we have a ticket to refactor getCsrfToken into middleware - we should do it like this ☝️